web_server/src/routes/mod.rs

```diff
@@ -30,3 +30,9 @@ pub(crate) async fn commit_transaction(transaction: Transaction<'_, Postgres>, m
     Ok(())
 }
 
+const EMAIL_TEMPLATE_PATH: &str = "./../templates/email.html";
+const QR_TEMPLATE_PATH: &str = "./../templates/qr.html";
```

Comment:
It is unnecessary to create such constants, move base path `./../templates` to `render_template` and use just template file names